Source/ARTIncrementalBackoffAndJitterRetryDelayCalculator.m

```diff
@@ -0,0 +1,38 @@
+#import "ARTIncrementalBackoffAndJitterRetryDelayCalculator.h"
+#import "ARTJitterCoefficientGenerator.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ARTIncrementalBackoffAndJitterRetryDelayCalculator ()
```

Comment:
For the sake of balance between "to be obvious" and the width of the screen to accommodate all these lengthy words I would just drop it to "ARTIncrementalRetryDelayCalculator"

Reply:
Makes sense, will change it.

Reply:
Changed it to `ARTBackoffRetryDelayCalculator` which I think best captures the essence of what this class does.